packages/design-system/src/Popover/Popover.mdx

```diff
@@ -0,0 +1,64 @@
+import { Canvas, Meta } from '@storybook/blocks';
+
+import * as PopoverStories from './Popover.stories';
+
+<Meta of={PopoverStories} />
+
+# Popover
+
+A Popover is a page overlay triggered by a button that displays additional interactive content.
+<Canvas of={PopoverStories.PopoverStory} />
+
+## Anatomy
+
+![Popover anatomy](./docs/popover-anatomy.png)
+
+1. **Popover panel**: Container for the popover content.
+2. **Header**: Contains an optional heading and an optional close icon.
+3. **Title**: A title is an optional element in the popover header that provides a brief label for the content inside the popover.
+4. **Close icon**: The close icon indicates to the user that the popover can be closed, either via clicking on the close icon or via pressing the `Esc` key.
+
+Consider that the close icon and title are not mandatory and might not be needed in every scenario. The inclusion and implementation of these features could differ based on the requirements of the use case.
+
+## Spacing
+
+![Popover spacing](./docs/popover-spacing.png)
+
+## Size
+
+### Small
+
+<Canvas of={PopoverStories.PopoverContentStory} />
+
+### Medium
+
+<Canvas of={PopoverStories.MediumPopoverContentStory} />
+
+## Usage
```

Comment:
Size usage is missing

Reply:
@vasanthappsmith Can you please help me with usage guidelines for the different sizes of the popover?

Reply:
https://github.com/appsmithorg/appsmith/issues/26285